client/app/pages/queries/components/SchemaBrowser.jsx

```diff
@@ -174,10 +204,10 @@ export default function SchemaBrowser({ schema, onRefresh, ...props }) {
 
 SchemaBrowser.propTypes = {
   schema: PropTypes.arrayOf(SchemaItemType),
-  onRefresh: PropTypes.func,
+  onRefresh: PropTypes.func
```

Comment:
comma after last object/array member was required (as it helps to keep diffs cleaner)